tests/foreman/ui/test_errata.py

```diff
@@ -732,8 +732,8 @@ def test_positive_filtered_errata_status_installable_param(session, errata_statu
             _set_setting_value(errata_status_installable, True)
             expected_values = {
                 'Status': 'OK',
-                'Errata': 'All errata applied',
-                'Subscription': 'Fully entitled',
+                'Errata': 'All errata applied clear',
+                'Subscription': 'Fully entitled clear',
```

Comment:
Do we expect "clear"? As in, is this property that we explicitly check and care about, or did it just happen to appear and we want test to continue to pass, as this is non-critical change?

I wonder if it would be better to change `actual_values` to drop "clear" from string before assertion. It would probably be best to use regex for that (since we want "clear" at end of string), and readability would mandate you to rewrite dict comprehension to normal loop.

And it probably would be "best" to change Airgun itself, so it understands there's this widget with new element and not only return label correctly, but also exposes action to click "clear", just in case we ever want to do that. But looking at code, it would require you to subclass `SatTableWithUnevenStructure` and write this new widget. At least that's what I would do. And it's probably not worth the time investment.

Reply:
> Do we expect "clear"? As in, is this property that we explicitly check and care about, or did it just happen to appear and we want test to continue to pass, as this is non-critical change?
> 
Its a new "feature" and I find the UI design aspects a little surprising.

> I wonder if it would be better to change `actual_values` to drop "clear" from string before assertion. It would probably be best to use regex for that (since we want "clear" at end of string), and readability would mandate you to rewrite dict comprehension to normal loop.
> 
OK, I can do some Re.match RegEx

> And it probably would be "best" to change Airgun itself, so it understands there's this widget with new element and not only return label correctly, but also exposes action to click "clear", just in case we ever want to do that. But looking at code, it would require you to subclass `SatTableWithUnevenStructure` and write this new widget. At least that's what I would do. And it's probably not worth the time investment.

I agree, there are stubbed tests and new features to write tests for before 6.8

Thank you